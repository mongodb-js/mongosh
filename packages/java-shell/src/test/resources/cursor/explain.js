// before
db.coll.remove({});
db.coll.insert({"_id": 1, name: "Vasya"});
db.coll.insert({"_id": 2, name: "Petya"});
db.coll.insert({"_id": 3, name: "Lyusya"});
// command extractProperty=queryPlanner extractProperty=namespace
db.coll.explain().find();
// command extractProperty=queryPlanner extractProperty=namespace
db.coll.explain("executionStats").find();
// command extractProperty=queryPlanner extractProperty=namespace
db.coll.find().explain();
// command extractProperty=queryPlanner extractProperty=namespace
db.coll.find().explain("executionStats");
// clear
db.coll.drop();