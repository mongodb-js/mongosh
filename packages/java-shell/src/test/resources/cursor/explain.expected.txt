"admin.coll"
"admin.coll"
"admin.coll"
"admin.coll"